ios/brave-ios/Sources/BraveWallet/WalletStrings.swift

```diff
@@ -563,6 +563,20 @@ extension Strings {
       value: "Deposit assets",
       comment: "The description of a swap button on the buy/send/swap/deposit modal"
     )
+    public static let bridge = NSLocalizedString(
```

Comment:
all the bridge stuff should be separated out

Reply:
reverted the last commit